canopen/objectdictionary/__init__.py

```diff
@@ -50,7 +50,7 @@ def export_od(od, dest: Union[str, TextIO, None] = None, doc_type: Optional[str]
             from canopen.objectdictionary import eds
             return eds.export_dcf(od, dest)
         else:
-            raise NotImplementedError(f"No support for the {doc_type!r} format")
+            raise ValueError(f"No support for the {doc_type!r} format")
```

Comment:
Consider also providing a more helpful error message.

```suggestion
            allowed = ", ".join(doctypes)
            raise ValueError(f"Cannot export to the {doc_type!r} format; allowed formats: {allowed}")
```

If you agree, let's do a similar change for the import API.

Reply:
Alternatively:

```suggestion
            allowed = ", ".join(doctypes)
            raise ValueError(f"Cannot export to the {doc_type!r} format; supported formats: {allowed}")
```

Reply:
Sure, go ahead.  Exception messages should be as helpful as possible.

Reply:
Addressed in 410b683de521255d6f79992285cf2f7dfa8fc831. Marking as resolved.